packages/mobile/test/schemas.ts

```diff
@@ -792,6 +792,12 @@ export const v19Schema = {
     },
     totalBalance: '40',
   },
+  app: {
+    ...v18Schema.app,
+    multiTokenShowHomeBalances: false,
+    multiTokenUseSendFlow: false,
+    multiTokenUseUpdatedFeed: false,
+  },
```

Comment:
nit: I'd prefer we bumped the schema because these properties are new.
As a rule we should increase the schema version every time we deviate from last schema published in an official release.
This avoids making wrong assumption for migrations as to what properties were part of a specific release.